scrapers/TeamskeetAPI.py

```diff
@@ -39,63 +33,77 @@ def save_json(api_json, url):
+
+
+# Not necessary but why not ?
+USER_AGENT = 'Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:79.0) Gecko/20100101 Firefox/79.0'
+
+
 fragment = json.loads(sys.stdin.read())
 if fragment["url"]:
     scene_url = fragment["url"]
 else:
-    debug('You need to set the URL (e.g. teamskeet.com/movies/*****)')
+    log.error('You need to set the URL (e.g. teamskeet.com/movies/*****)')
     sys.exit(1)
 
 if "teamskeet.com/movies/" not in scene_url:
-    debug('The URL is not from a Teamskeet URL (e.g. teamskeet.com/movies/*****)')
+    log.error('The URL is not from a Teamskeet URL (e.g. teamskeet.com/movies/*****)')
     sys.exit(1)
 
 scene_id = re.sub('.+/', '', scene_url)
 if not scene_id:
-    debug("Error with the ID ({})\nAre you sure that the end of your URL is correct ?".format(scene_id))
+    log.error("Error with the ID ({})\nAre you sure that the end of your URL is correct ?".format(scene_id))
     sys.exit(1)
 use_local = 0
 json_file = os.path.join(DIR_JSON, scene_id+".json")
 if os.path.isfile(json_file):
-    print("Using local JSON...", file=sys.stderr)
+    log.debug("Using local JSON...")
     use_local = 1
     with open(json_file, encoding="utf-8") as json_file:
         scene_api_json = json.load(json_file)
 else:
-    print("Asking the API...", file=sys.stderr)
+    log.debug("Asking the API...")
     api_url = 'https://store2.psmcdn.net/ts-elastic-d5cat0jl5o-videoscontent/_doc/{}'.format(
         scene_id)
     headers = {
         'User-Agent': USER_AGENT,
         'Origin': 'https://www.teamskeet.com',
         'Referer': 'https://www.teamskeet.com/'
     }
-
+    scraper = cloudscraper.create_scraper()
     # Send to the API
     r = ""
     try:
-        r = requests.get(api_url, headers=headers, timeout=(3, 5))
+        r = scraper.get(api_url, headers=headers, timeout=(3, 5))
     except requests.exceptions.RequestException:
-        print("An error has occurred with Requests", file=sys.stderr)
-        print(f"Request status: `{r.status_code}`", file=sys.stderr)
-        print(f"Check your TeamskeetJSON.log for more details", file=sys.stderr)
+        log.error("An error has occurred with Requests")
+        log.error(f"Request status: `{r.status_code}`")
+        log.error(f"Check your TeamskeetJSON.log for more details")
         with open("TeamskeetAPI.log", 'w', encoding='utf-8') as f:
             f.write("Scene ID: {}\n".format(scene_id))
             f.write("Request:\n{}".format(r.text))
         sys.exit(1)
-    scene_api_json_check = r.json()['found']
-    if scene_api_json_check:
-        scene_api_json = r.json()['_source']
-    else:
-        debug('Scene not found (Wrong ID?)')
+    try:
+        scene_api_json_check = r.json().get('found')
+        if scene_api_json_check:
+            scene_api_json = r.json()['_source']
+        else:
+            log.error('Scene not found (Wrong ID?)')
+            sys.exit(1)
+
+    except:
+        log.debug(f"invalid input {r.content}")
```

Comment:
```suggestion
        if "Please Wait... | Cloudflare" in r.text:
            log.error("Protected by Cloudflare.")
        else:
            log.debug(f"invalid input {r.content}")
```
I am not a fan of printing the whole html into the log.

Reply:
yikes..yes i only used that to see why the scraper was blocking, i was going to remove that later on.
should we skip the `r.content` print in the else also and leave a simple `log.error("invalid input")` ?

Reply:
I would keep the `log.error("invalid input")`